eslint.js

```diff
@@ -336,7 +336,7 @@ module.exports = {
         'prefer-arrow-callback': 'error',
         'prefer-const': 'error',
         'prefer-destructuring': ['error', {
```

Comment:
I thought we would just turn the whole rule off, I think object has the same problems